ext/drcovlib/drcovlib.c

```diff
@@ -213,7 +213,7 @@ static void
 dump_drcov_data(void *drcontext, per_thread_t *data)
 {
     version_print(data->log);
-    module_table_print(module_table, data->log, false);
+    module_table_print(module_table, data->log, options.verbose_module_table);
```

Comment:
In general, we should avoid any format change if possible.
In this case, we did not officially publish the drcov format, and only support drcov2lcov to read it, so it is less risky to change the format.
I think it should be fine to append a few fields after the path.